crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -649,10 +650,27 @@ fn is_exception_or_base_exception(qualified_name: &QualifiedName) -> bool {
     )
 }
 
+fn starts_with_returns(docstring: &Docstring) -> bool {
+    if let Some(first_word) = docstring.body().as_str().split(' ').next() {
+        return first_word == "Returns";
+    }
```

Comment:
We should also check for "Return" (no 's') according to the docs (https://google.github.io/styleguide/pyguide.html#doc-function-returns)

> It may also be omitted if the docstring starts with “Return”, “Returns”, “Yield”, or “Yields”

Reply:
Ya was gonna do that as well